_inc/connect-button.js

```diff
@@ -72,7 +72,7 @@ jQuery( document ).ready( function( $ ) {
 		},
 		handleAuthorizationComplete: function() {
 			jetpackConnectButton.isRegistering = false;
-			location.reload();
+			window.location.hash = '#/plans-prompt';
```

Comment:
Should we redirect to a different screen if the user has already purchased a plan?

Reply:
@jeffgolenski what do you think?

Reply:
Okay, I'll look for some way to achieve this.

Reply:
This has been addressed in https://github.com/Automattic/jetpack/pull/13348/commits/8ec4c975d06655b38dae7304ce78c03b4329a5f7.